atlas/atlas.md

```diff
@@ -0,0 +1,77 @@
+# Atlas Package Cloner
+
```

Comment:
atlas dir should be moved under tools/atlas

Reply:
"nimpretty", "drnim" and "nimsuggest" are not under "tools" either. I think the rule is pretty much "if the tool is a single file, it goes under tools/ otherwise it gets its own directory".

Reply:
scoping is still a good idea in 2021 though... every time we add something at the top-level it makes scope pollution worse.

## example 1
this works but shouldn't; I install compiler but it adds something else in scope:
```
nimble install compiler
nim r -f --eval:'import nimsuggest/sexp'
```

## example 2
it's even worse if I install compiler pkg with with `nimble develop`, it puts everything in top-level scope:
```
cd ~
nimble develop /pathto/Nim
this works but shouldn't:
nim r --eval:'import atlas/osutils'
```